api/proto/metrics/metrics.proto

```diff
@@ -173,6 +173,60 @@ message appCpuMetric {
   uint64 total_ns = 7;                 // like 'total' but in nanoseconds. Overflow may happen after ~40 years without EVE restart
 }
 
+message SmartAttr {
+  uint32 id = 1;
+  string name = 2;
+  uint64 value = 4; // the current value of the parameter
+  uint64 worst = 5; // worst value that Value has ever reached
+  uint64 thresh = 6;  // the value that the Value of the same attribute must reach for the attribute's state to be considered critical.
+  string when_failed = 9; // critical state if no empty
+  string raw_value = 10;
+}
+
+message ValueSelfTest {
+  uint32 num = 1;
+  string name = 2;
+  uint32 status = 3;
+  string remaining = 4;
+  uint64 lifetime = 5;  //in hours
+  string lba_of_first_error = 6;
+}
+
+message SmartMetric {
+  SmartAttr reallocated_sector_ct = 1;
+  SmartAttr power_on_hours = 2;
+  SmartAttr power_cycle_count = 3;
+  SmartAttr reallocated_event_count = 4;
+  SmartAttr current_pending_sector = 5;
```

Comment:
Are these the only we care about to determine if solid state and/or rotating disk storage is aging and going bad?

Reply:
In fact, we can get any values we need from smartctl, but for a start we decided to take only these 5 values for now. If you think other values can be added, please let me know.
Also, if there are will be a lot of values, it can be converted into a slice with attributes.